emily/handler/src/database/accessors.rs

```diff
@@ -201,7 +202,10 @@ pub async fn pull_and_update_deposit_with_retry(
         if update.is_unnecessary(&deposit_entry) {
             return Ok(deposit_entry);
         }
-        if !is_trusted_key && deposit_entry.status != Status::Pending {
+        let is_valid_untrusted_status_update = update.event.status == StatusEntry::Accepted
+            && (deposit_entry.status == Status::Pending
+                || deposit_entry.status == Status::Accepted);
+        if !is_trusted_key && !is_valid_untrusted_status_update {
```

Comment:
IIUC this enables overwriting an accepted entry with different accepted info (eg, the stacks block). Why do we want this? vs keeping that updating an accepted deposit with accepted is forbidden, and just not bail in the caller

Reply:
AFAIU requests are `Accepted` before the transaction is confirmed, and if this tx got RBFd we want to make update `Accepted` -> `Accepted`.

Reply:
oh you're right, in case of sweep rbf we want this to be updated.